packages/shared/src/lib/featureManagement.ts

```diff
@@ -29,6 +29,7 @@ const feature = {
   bookmarkReminder: new Feature('bookmark_reminder', false),
   animatedUpvote: new Feature('animated_upvote', false),
   mobileExploreTab: new Feature('mobile_explore_tab', false),
+  onboardingContentType: new Feature('onboarding_content_type', false),
```

Comment:
In growthbook https://app.growthbook.io/features/onboarding_content_type and already enabled for team